app/helpers/pageflow/config_helper.rb

```diff
@@ -27,5 +27,36 @@ def config_file_model_types_seed(json, config)
         json.set!(collection_name, file_type.model.name)
       end
     end
+
+    # Render seed data that can be used to build share urls.
+    #
+    # @param [JBuilder] json
+    # @param [Entry] entry
+    # @since edge
+    def config_sharing_data_seed(json, entry)
+      json.share_url pretty_entry_url(entry)
+      json.share_url_templates Pageflow::SocialShareLinksHelper::PROVIDER_URL_TEMPLATES
+    end
+
+    # Render seed data that can be used to display legal information.
+    #
+    # @param [JBuilder] json
+    # @param [Entry] entry
+    # @since edge
+    def config_imprint_and_privacy_seed(json, entry)
+      theming = entry.theming
+      json.imprint do
+        json.imprint_link_label theming.imprint_link_label
+        json.imprint_link_url theming.imprint_link_url
+      end
+      json.copyright do
+        json.copyright_link_label raw(theming.copyright_link_label)
+        json.copyright_link_url theming.copyright_link_url
+      end
+      json.privacy do
+        json.privacy_link_label I18n.t('pageflow.public.privacy_notice')
+        json.privacy_link_url "#{theming.privacy_link_url}?lang=#{entry.locale}"
+      end
+    end
```

Comment:
I'm on the fence if this really carries its weight as a separate helper. Since this will only be used by the scrolled entry seed helper, it might as well be part of the JSON template. In it's current state it also duplicates most of the logic in `Pageflow::EntriesHelper#entry_global_links`. So one could also consider instead only extracting a helper for the privacy link url to prevent building that in two places. Is therea reason why the copyright link label is passed `raw` while imprint link label isn't?

Also, helpers used across engine boundaries should have a spec.